task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +343,7 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    return num % 9
```

Comment:
Try to solve this function in a different way, based on the description. Thank you!